src/app/dynamic_server/DynamicDispatcher.cpp

```diff
@@ -412,3 +412,15 @@ const EmberAfAttributeMetadata * emberAfLocateAttributeMetadata(EndpointId endpo
     // no known attributes even for OTA
     return nullptr;
 }
+
+Protocols::InteractionModel::Status emberAfReadAttribute(unsigned short, unsigned int, unsigned int, unsigned char *,
```

Comment:
Why was this change needed?  What's building DynamicDispatcher _and_ using these functions, exactly?

This feels like it should have been in a separate PR (along with whatever is actually pulling this bit in), so it can be reviewed without all the other noise here.

Reply:
Thank you for your valid review. This was added for dynamic server, However, we don't need this anymore. so, I have deprecated these changes.